exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -864,6 +872,14 @@ public String getDigest() {
     return toString();
   }
 
+  public void setCacheFileRoot(String cacheFileRoot) {
+    this.cacheFileRoot = cacheFileRoot;
+  }
+
+  public void setBatchSize(long batchSize) {
+    this.recommendedBatchSize = batchSize;
```

Comment:
I feel it's fine we do not have such check.  Max value for option is checked when users tries "alter session set" command. 

For "limit n" query where n is > max value, ParquetRecordReader would pick min of two values, essentially ignore this recommendedBatchSize (?).